.editorconfig

```diff
@@ -2,6 +2,6 @@
 root = true
```

Comment:
I used this configuration because, in a previous PR, my Goland used 4 spaces and changed multiple lines I didn't touch.
I removed the file, but I can't see the linting job. Let me know if the error persists.

About which editor supports it, the most popular ones do without any plugin: VS Code, IntelliJ, Vim, etc. It's an [open-source standard](https://editorconfig.org/)